ledger/participant-integration-api/src/test/lib/scala/platform/store/dao/DbDispatcherTests.scala

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.platform.store.dao
```

Comment:
I found it discomforting that:
* We are testing an implementation independent non-db functionality with a DB test: can't we just roll a unit test for the additional functionality instead? (without DB)
* What happens if those PartyRecord stuff changes? Or removes? Shouldn't a DbDispatcher test be oblivious to that?

Reply:
removed that test ;) it will be covered then later in another PR with functionality which actually using this method